exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -115,6 +115,8 @@
   private List<RowGroupInfo> rowGroupInfos;
   private Metadata.ParquetTableMetadataBase parquetTableMetadata = null;
   private String cacheFileRoot = null;
+  private int batchSize;
+  private static final int DEFAULT_BATCH_LENGTH = 256 * 1024;
```

Comment:
They have the same value because it is the default record batch size value we use in parquet reader (DEFAULT_BATCH_LENGTH  in ParquetRecordReader).  

Yes, new option is honored. If you look at changes in the parquetRecordReader, the value we pick for batch size is the minimum of  (defaultSize, batchSize coming from limit, user configured value).